### DISCUSSION TITLE
用Alist链接另一个Alist无法上传文件，

### DISCUSSION THREAD

**Discussion by fangfang333** [AlistGo/alist#417] [Q&A]
问题一：用alist链接另一个alist，上传按钮会消失，没有任何元数据，应该不是权限的问题，改用webdav互相连接无问题。
问题二：添加元信息对路径增加密码保护之后，会导致webdav无法打开那个文件夹，网页版可以正常访问。
![1](https://user-images.githubusercontent.com/78396305/150641780-0b7ec857-908c-4365-b3ba-60fe60a56733.jpg)
![2](https://user-images.githubusercontent.com/78396305/150641784-09822fba-175d-4095-b49c-52cc67cad55b.jpg)
![3](https://user-images.githubusercontent.com/78396305/150641785-cdff5eb6-8a4b-47a8-a6e4-3fb0672d04f1.jpg)
![4](https://user-images.githubusercontent.com/78396305/150641788-5d92ba01-167f-46b4-9b0d-9d94f3b2330f.jpg)

**Comment by xhofe** [comment]
1. 本来就不支持，仅用于浏览  https://alist-doc.nn.ci/docs/webdav
2. 测试无问题，可以正常打开

**Reply by fangfang333** [reply]
好的，感谢解答，第二个问题我又测试了一遍，没报错了。那请问有考虑过以后增加多个alist之间访问的更多功能吗，如果多个alist之间可以进行更多交互操作，就可以实现一个面板管理多台机器的文件了，还能管理多个网盘里的文件和S3存储的文件，真的太赞了。

**Reply by fangfang333** [reply]
因为有些网盘是国外的，有些网盘是国内的，如果我只有一台机器，我不知道这台机器应该买国外的还是国内的，有点尴尬，所以我现在 在多台机器上都安装了alist。

**Reply by xhofe** [reply]
上传由服务器中转，套娃之后两个会 客户端->alist1服务器->alist2服务器->网盘，我觉得这么做多此一举。新建文件夹/重命名什么的倒是可以加。

**Reply by fangfang333** [reply]
可以通过路径来解决套娃的问题，不会循环读取(除非故意的)，我的意思就是想实现这种统一管理的方式
![Z}G` FKF3IJ} {$S)$}MF60](https://user-images.githubusercontent.com/78396305/150643717-de739d1b-54fe-42d8-a98b-e602ca964bf7.png)
